shop/src/test/java/ro/msg/learning/shop/strategyTest/UnitTestStrategy.java

```diff
@@ -0,0 +1,46 @@
+package ro.msg.learning.shop.strategyTest;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import ro.msg.learning.shop.model.Customer;
+import ro.msg.learning.shop.model.Location;
+import ro.msg.learning.shop.model.ProductOrder;
+import ro.msg.learning.shop.repository.CustomerRepository;
+import ro.msg.learning.shop.repository.ProductOrderRepository;
+import ro.msg.learning.shop.repository.StockRepository;
+import ro.msg.learning.shop.service.LocationService;
+import ro.msg.learning.shop.service.StockService;
+
+import java.time.LocalDateTime;
+
+@ExtendWith(MockitoExtension.class)
+public class UnitTestStrategy {
```

Comment:
class name is the same as the class you are testing, it just has "test" at the end
- make as many classes as you need if u need to test more methods from more classes :) they are free